src/GraphQL/TypeExtensions.cs

```diff
@@ -206,9 +211,7 @@ internal static string GetFriendlyName(this Type type)
         /// </summary>
         public static Type GetEnumerableElementType(this Type type)
```

Comment:
This is only ever used by `GetTypeFromType`

Reply:
Obsolete?